src/components/AddView.js

```diff
@@ -1,10 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
 
 const AddView = () => {
+  const [state, setState] = useState({
+    purchaseItem: '',
+    howSoon: '7',
+    lastPurchaseDate: null,
+  });
+
+  const handleChange = (e) => {
+    setState({
+      ...state,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (state.purchaseItem.trim()) {
+      let token =
+        localStorage.length !== 0 ? localStorage.getItem('token') : 'LA';
+      db.collection(token)
+        .add({
+          purchaseItem: state.purchaseItem,
+          howSoon: state.howSoon,
+          lastPurchaseDate: state.lastPurchaseDate,
+        })
+        .then(() => {
+          alert('Purchase Item has been recorded');
+        })
+        .catch((error) => {
+          alert(error.message);
+        });
+
+      setState({
+        purchaseItem: '',
+        howSoon: '7',
+        lastPurchaseDate: null,
+      });
+    } else {
+      alert('Please write an item');
+    }
+  };
+
   return (
-    <main>
-      <h1>Add Item</h1>
-    </main>
+    <>
+      <form className="form-container" onSubmit={handleSubmit}>
+        <label htmlFor="purchaseItem">Purchase Item Name:</label>
+        <input
+          type="text"
+          name="purchaseItem"
+          value={state.purchaseItem}
+          onChange={handleChange}
+        />
+
+        <label htmlFor="howSoon">
+          How soon are you likely to buy it again?:<br></br>
+        </label>
+        <select
```

Comment:
Just like with a text input, we  need to add an id to our ```<select>``` and make sure that the ```for``` attribute on the label matches the id value on the ```<select>```.
And again..  for future styling of the select, it’d be helpful to wrap native ```<select>``` in a ```<div>```

Reply:
Thank you @katia-utochkina. Why do we need to wrap it? It looks like I can still style without it

Reply:
it's how it's usually done when it comes to form styling. Up to you.